Misc/NEWS.d/next/Library/2023-11-11-16-42-48.gh-issue-109538.cMG5ux.rst

```diff
@@ -0,0 +1 @@
+Issue warning message instead of having :class:`RuntimeError` be displayed when event loop has already been closed at :method:`StreamWriter.__del__`
```

Comment:
To fix `../build/NEWS:159: ERROR: Unknown interpreted text role "method".` on the CI:

```suggestion
Issue warning message instead of having :class:`RuntimeError` be displayed when event loop has already been closed at :meth:`StreamWriter.__del__`.
```